client/src/components/Settings/SettingsScreen.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useState } from 'react';
+import {SafeAreaView,
+    Text,
+    StyleSheet,
+    View,
+    ScrollView,
+    FlatList,
+    Switch,
+    TouchableOpacity,
+    Image,
+    Dimensions,
+
+} from 'react-native';
+
+
+// List of settings items
+const Sections = [
+    {
+        header: 'Notications',
+        items: [
+            {id: 'notifiyNewMessage', title: 'Notifications for new messages', type: 'toggle'},
+        ]
+    },
+    {
+        header: 'Preferences',
+        items: [
+            {id: 'darkMode', title: 'Dark Mode', type: 'toggle'},
+            {id: 'language', title: 'Language', type: 'select'},
+        ]
+    },
+    {
+        header: 'Privacy',
+        items: [
+            {id: 'deleteMessages', title: 'Delete messages', type: 'toggle'},
+        ]
+    },
+
+]
+
+// props to pass into settings
+type SettingsProps = {
+    id: string,
+    title: string,
+    type: string,
+}
+
+
+
+const SettingsScreen : React.FC = () => {
+
+    // settings values (will be changed later to reflect the actual settings)
+    const [data, setData] = useState({
+        notifyNewMessage: true,
+        darkMode: false,
+        language: 'English',
+        deleteMessages: false,
+        
+    });
+
+    // The component for a row that contains a single setting
+    const SettingsRow = ({id, title, type}: SettingsProps) => (
```

Comment:
Can you move this into a file? Possibly within the component folder? Could also be in this folder, just to make it cleaner :)